app/services/analytics/ecf_school_service.rb

```diff
@@ -40,6 +40,7 @@ def update_school_analytics
                 lead_provider: school.active_partnerships.first&.lead_provider&.name || school.partnerships.first&.lead_provider&.name,
                 delivery_partner: school.active_partnerships.first&.delivery_partner&.name || school.partnerships.first&.delivery_partner&.name,
                 chosen_cip: school.school_cohorts.find { |sc| sc.cohort_id == current_cohort_id }&.core_induction_programme&.name,
+                active_participants: school.includes(:school_cohorts).where(cohort: current_cohort_id)&.active_ecf_participants&.any?,
```

Comment:
```suggestion
                active_participants: school.school_cohorts.find { |sc| sc.cohort_id == current_cohort_id }&.active_ecf_participants&.any?,
```

and also change above to 
```ruby
        School.eligible_or_cip_only.includes(
          :nomination_emails,
          :induction_coordinators,
          :partnerships,
          school_cohorts: :active_ecf_participants,
          active_partnerships: %i[lead_provider delivery_partner],
        ).find_in_batches do |schools|
```